File: php/Car.php
<?php

require_once "Vehicle.php";

class Car extends Vehicle {
    private $humanCapacity;
    private $numberOfDoors;

    // Constructor
    public function __construct($plate, $brand, $year, $clr, $capacity, $doors) {
        parent::__construct($plate, $brand, $clr, $year);
        $this->humanCapacity = $capacity;
        $this->numberOfDoors = $doors;
    }

    // Getters
    public function getHumanCapacity() {
        return $this->humanCapacity;
    }

    public function getNumberOfDoors() {
        return $this->numberOfDoors;
    }

    // Setters
    public function setHumanCapacity($capacity) {
        $this->humanCapacity = $capacity;
    }

    public function setNumberOfDoors($doors) {
        $this->numberOfDoors = $doors;
    }
}

File: java/Car.java
public class Car extends Vehicle {
    private int humanCapacity;
    private int numberOfDoors;

    // Constructor
    public Car(String plate, String brand, int year, String clr, int capacity, int doors) {
        super(plate, brand, clr, year);
        this.humanCapacity = capacity;
        this.numberOfDoors = doors;
    }

    // Getters
    public int getHumanCapacity() {
        return humanCapacity;
    }

    public int getNumberOfDoors() {
        return numberOfDoors;
    }

    // Setters
    public void setHumanCapacity(int capacity) {
        this.humanCapacity = capacity;
    }

    public void setNumberOfDoors(int doors) {
        this.numberOfDoors = doors;
    }
}
